core/src/backends/cln/grpc/node.rs

```diff
@@ -68,6 +70,25 @@ impl NodeMethods for ClnGrpc {
         Ok(result)
     }
 
+    async fn get_invoice(&self, payment_hash: String) -> Result<Invoice, Error> {
+        let mut client = self.get_client().await?;
+
+        let payment_hash_hex = match hex::decode(payment_hash) {
+            Ok(hex) => hex,
+            Err(_) => return Err(Error::ApiError(String::from("Invalid payment_hash"))),
+        };
```

Comment:
You can remove the `match` and reduce it to this:
```suggestion
        let payment_hash_hex = hex::decode(payment_hash).map_err(|_| Error::ApiError(String::from("Invalid payment_hash")))?;
```